src/parser/analyze-type/index.ts

```diff
@@ -53,7 +53,12 @@ function appendDeclareSvelteVarsTypesFromAST(
           node.label.name !== "$" ||
           parent !== result.ast ||
           node.body.type !== "ExpressionStatement" ||
-          node.body.expression.type !== "AssignmentExpression"
+          node.body.expression.type !== "AssignmentExpression" ||
+          (node.body.expression.left.type !== "Identifier" &&
```

Comment:
I recommend to add some comments here, or at least link the original issue?

Reply:
Thank you for your review!
I added comments.